tools/distpartitioning/parmetis_preprocess.py

```diff
@@ -287,6 +287,36 @@ def gen_parmetis_input_args(params, schema_map):
         schema_map[constants.STR_NUM_NODES_PER_CHUNK],
     )
 
+    # Check if <graph-name>_stats.txt exists, if not create one using metadata.
+    # Here stats file will be created in the current directory. 
+    # No. of constraints, third column in the stats file is computed as follows:
+    #   num_constraints = no. of node types + train_mask + test_mask + val_mask
+    #   Here, (train/test/val) masks will be set to 1 if these masks exist for
+    #   all the node types in the graph, otherwise these flags will be set to 0
+    assert graph_name in schema_map, "Graph name is not present in the json file"
```

Comment:
where is `graph_name` defined? should be `constants.GRAPH_NAME`?

Reply:
graph_name is present in the json file of the input graph dataset. It is one of the keys in the dictionary object created by reading the json file. Per our new dataset format, graph_name should be present in the json file for the input dataset. 

This is a typo and was interchanged the next line. You are right, it should be constants.GRAPH_NAME. Changed it accordingly.